migration/v0.19.x-v0.20.x/upgrade-apps.md

```diff
@@ -8,29 +8,31 @@
 
 2. Run the migration script: `move-alertmanager-groupby.sh`
 
-3. Update apps configuration:
+3. Run the migration script: `move-usergroups-user-grafana.sh`
```

Comment:
Can you change all of the numbering to `1.`? 
That way we don't have to updated the ordering every time another step is needed.